### PR TITLE
Fix test runners integration mess

### PR DESCRIPTION
**Pull request by webknjaz**
* Add pytest-runner integration instead of self-crafted class
* Drop tox from requirements as nobody uses it (there's even no `tox.ini` present)

Includes #145

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -65,14 +64,8 @@ def read(f):
...
 
-class PyTest(TestCommand):
-    user_options = []
-
-    def run(self):
-        import subprocess
-        errno = subprocess.call([sys.executable, '-m', 'pytest', 'tests'])
-        raise SystemExit(errno)
-
+NEEDS_PYTEST = {'pytest', 'test'}.intersection(sys.argv)
```
Comment (asvetlov) [inline setup.py]: How it works?

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[aliases]
+test = pytest
```
Comment (asvetlov) [inline setup.cfg]: Please no.
I pretty happy with `pytest` name.

File: setup.py (hunk 1 above)
Comment (webknjaz) [reply setup.py]: this just checks for convenient argument at adds pytest-runner to setup_requires only if it's needed, meaning it won't install `pytest-runner` if you just do `sdist` or `bdist_wheel`

File: setup.cfg (hunk 2 above)
Comment (webknjaz) [reply setup.cfg]: It's an official way of integration + it worked this way before: https://github.com/aio-libs/multidict/pull/146/files/9e271b8286eda8237ad569495095c2b3793d81bc#diff-2eeaed663bd0d25b7e608891384b7298L104

Comment (webknjaz) [reply setup.cfg]: AFAIK you don't even use this

Comment (webknjaz) [reply setup.cfg]: I'd disable possibility for `python setup.py test` to invoke `unittest`

Comment (asvetlov) [reply setup.cfg]: Finally got it.

File: setup.py (hunk 1 above)
Comment (asvetlov) [reply setup.py]: But you never check the value, isn't it?

Comment (webknjaz) [reply setup.py]: Oops, it's intended to be checked on the next line, I'll fix this.

Comment (webknjaz) [reply setup.py]: @asvetlov done

**Pull request merged**